cmd/server/server.go

```diff
@@ -50,6 +50,7 @@ type serverArgs struct {
 	// mixer manager args
 	serviceConfigFile      string
 	globalConfigFile       string
+	kubeConfig             string
```

Comment:
Agree.  Can we quickly create an admin API that receives a full config file (or similar) and leave the polling to the manager?